### ISSUE TITLE
documentation missing extra # Doctrine Configuration

### ISSUE THREAD

**Issue by mansimas** [ongr-io/SettingsBundle#139]
http://ongr.readthedocs.org/en/latest/components/SettingsBundle/index.html
would be good to include 
```
# Doctrine Configuration
doctrine:
    dbal:
        server_version: "5.6" // need to include server version
```
without this, /settings_prefix/login drops exception:
```
An exception occured in driver: SQLSTATE[42000] [1049] Unknown database 'ongr' 
```

**Comment by martiis** [comment]
duplicate https://github.com/ongr-io/SettingsBundle/issues/100

**Comment by emaslak** [comment]
Closed in #140
